code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -58,6 +58,21 @@
 				//End bloody footprints
 
 				S.step_action()
+		
+	if(bleed_rate && !buckled && has_gravity(loc) && lying)
+		if(prob(2) && istype(loc, /obj/structure/closet))
+			visible_message("<span class='warning'>[src] bumps their head against \the [src]!</span>")

```

Comment:
You should probably check if they have a head in the first place.

Reply:
In addition, does this message reveal the identity of the lockered person for witnesses?
If it does, that's not something they should be able to know.